nemo_rl/models/generation/vllm/vllm_backend.py

```
@@ -273,3 +274,12 @@ def start_gpu_profiling(self) -> None:
     def stop_gpu_profiling(self) -> None:
         """Stop GPU profiling."""
         torch.cuda.profiler.stop()
+
+    def report_node_ip_and_gpu_id(self) -> list[tuple[str, int]]:
+        """Report the node IP and GPU ID of the current worker."""
+        ip = ray._private.services.get_node_ip_address()
+        try:
+            gpu_id = int(ray.get_gpu_ids()[0])
+        except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)